src/main/scala/com.snowplowanalytics.snowplow.sbt/SnowplowDockerPlugin.scala

```diff
@@ -24,7 +24,7 @@ object SnowplowDockerPlugin extends AutoPlugin {
 
   override def projectSettings: Seq[Setting[_]] = Seq(
     Docker / maintainer := "Snowplow Analytics Ltd. <support@snowplow.io>",
-    dockerBaseImage := "eclipse-temurin:11-jre-focal",
+    dockerBaseImage := "eclipse-temurin:21-jre-noble",
```

Comment:
Is there a reason why maintaining non-distroless images of our apps ?

Reply:
1. Open source users
2. Any of our apps which use hadoop: Lake Loader and S3 loader